### ISSUE TITLE
Datasets: create warning about calling clear_data_dir()

### ISSUE THREAD

**Issue by ElisovaIra** [maks-sh/scikit-uplift#180]
## 🐛 Bug

Dataset's loading mechanic is:
1) calling `fetch_[dataset name]()`
2) if dataset is present in `data dir` ->  `read` dataset from dir
3) if not: `download` dataset 

❗️but when there is a broken dataset file present in data dir with the correct name - there is no checker or warning about it and `fetch_[dataset name]()` is trying to read the wrong file and crashes 

## To Reproduce

1. Put an empty `lenta_dataset.csv.gz` file in `usr/scikit-uplift-data` dir
2. Run the code
```
from sklift.datasets import fetch_lenta
dataset = fetch_lenta()
```

How to fix:
```
from sklift.datasets import fetch_lenta, clear_data_dir
clear_data_dir()
dataset = fetch_lenta()
```

## Expected behavior

Create a checker for broken dataset 
or
warning to call clear_data_dir()

## Environment

 - scikit-uplift version (e.g., 0.1.2): 0.4.1
 - Python version (e.g., 3.7): 3.7
 - OS (e.g., Linux): MacOS

**Comment by maks-sh** [comment]
Great idea.

I think it can be implemented this way:

1) calculate the hash from the file once, 
2) hardcode it inside the function and then
3) compare it with the hash from the already downloaded file.
4) if the hashes are not equal then raise an exception

**Comment by maks-sh** [comment]
Implemented in 0.5.0
